### PR TITLE
Add support for --ssl-no-revoke

### PR DESCRIPTION
**Pull request by shayfisher**
Issue #47 

Add a new ENV var called COCOAPODS_ART_SSL_NO_REOVKE to be able to pass the flag --ssl-no-revoke when using curl with WinSSL

### CODE REVIEW

**Comment by shayfisher** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by shayfisher** [comment]
recheckcla

**Comment by shayfisher** [comment]
recheckcla

**Review by shayfisher — COMMENTED** [review]
.

File: lib/cocoapods_plugin.rb
Comment (sphanley) [inline lib/cocoapods_plugin.rb]: Not sure if you're still pursuing this PR, but for what it's worth, you've got a typo in `_REVOKE` here.

**Comment by eyalbe4** [comment]
@shayfisher - 
Looks like this PR is pending for too long! Sorry about that.
Would you mind documenting the new environment varoable in the README as well? We'll make sure to merge this right after you add it.

**Comment by sphanley** [comment]
Just a reminder that there's still a typo in the variable name (which has now been propagated to the README), someone should make sure that's fixed before the merge!

EDIT: Looks like maybe I didn't submit my original review comment correctly so it may not have shown up until now - so I guess this is a fresh FYI, not a reminder, hah.

File: README.md
Comment (sphanley) [inline README.md:39]: Should this be `export` rather than `set`?

Comment (shayfisher) [reply README.md:39]: It is only relevant on Windows which uses WinSSL for curl, therefore set is used (it is a windows command)

Comment (sphanley) [reply README.md:39]: Ah, that makes sense, it's been a long time since I've been a Windows CLI guy. Sorry for the bother!

Comment (shayfisher) [reply README.md:39]: Basically this flag would be rarely used as this art-cocoapods plugin is mainly about macOS development. But there are use cases such as ours where the security scanning for instance is being performed on an analyst's workstation which is windows based. He has all the tooling right now to scan Podfiles but he is unable to pull the Artifact since we are running on premise and access to CRL servers is blocked (no outbound traffic) so pulling pods with the plugin via HTTPS fails.
Hope the intentions here are clear enough.
Thank you for reviewing

Comment (sphanley) [reply README.md:39]: That's an interesting use case that I wouldn't have even thought of. Thanks for the explanation!

Comment (shayfisher) [reply README.md:39]: @eyalbe4 done

**Review by eyalbe4 — APPROVED** [review]

**Pull request merged**
